migrations/20210615074327-platform-tip-debt-kind.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      UPDATE "Transactions"
+      SET "kind" = 'PLATFORM_TIP_DEBT'
+      WHERE "kind" = 'PLATFORM_TIP'
+      AND "isDebt = TRUE
+    `);
+  },
+
+  down: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      UPDATE "Transactions"
+      SET "kind" = 'PLATFORM_TIP'
+      WHERE "kind" = 'PLATFORM_TIP_DEBT'
+    `);
+  },
+};
```

Comment:
Where are we adding `PLATFORM_TIP_DEBT` to the `enum_Transactions_kind`?

Reply:
It still WIP